### PR TITLE
add get pages content from sanity #CU-1xpwq7v

### PR DESCRIPTION
**Pull request by AndresRDC**
#{CU-1xpwq7v}

# Add get pages content from sanity

### CODE REVIEW

**Comment by hifrontendcafe** [comment]
Task linked: [CU-1xpwq7v [WEBSITE] Tomar las páginas con contenido desde sanity](https://app.clickup.com/t/1xpwq7v)

File: src/lib/queries.ts [hunk 1]
```diff
@@ -13,6 +13,17 @@ export const settingsQuery = groq`
...
+export const pageQueryByHero = groq`
+  *[_type == "page" && hero == $hero][0]{
+    hero,
+    title,
+    'shortDescription': coalesce(shortDescription, ''),
+    'description': coalesce(description, ''),
+    'doc': coalesce(doc, ''),
+    metadata
+  }
+`;
```
Comment (Santiago-j-s) [inline src/lib/queries.ts:25]: I didn't know that groq has coalescing for strings, great!

**Review by Santiago-j-s — APPROVED** [review]
Great job! LGTM

File: src/components/FecHead/index.tsx [hunk 2]
```diff
@@ -1,31 +1,41 @@
...
+      {!hasMetadataTitle && (
+        <meta property="og:title" content={`${title} - FrontendCafé `} />
+      )}
+      {!hasMetadataDescription && (
+        <meta property="og:description" content={description} />
+      )}
+      {!hasMetadataImage && <meta property="og:image" content={ogImage} />}
+      {metadata.map((meta: Metadata) => (
+        <meta key={meta._key} property={meta.property} content={meta.content} />
+      ))}
```
[the remark this answers is not in the document]
Comment (Santiago-j-s) [reply src/components/FecHead/index.tsx:38]: Ok, no había entendido, esta perfecto

**Pull request merged**